staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -225,15 +226,15 @@ func (f *DeltaFIFO) KeyOf(obj interface{}) (string, error) {
 }
 
 // HasSynced returns true if an Add/Update/Delete/AddIfNotPresent are called first,
-// or an Update called first but the first batch of items inserted by Replace() has been popped
+// or the first batch of items inserted by Replace() has been popped。
 func (f *DeltaFIFO) HasSynced() bool {
 	f.lock.Lock()
 	defer f.lock.Unlock()
 	return f.populated && f.initialPopulationCount == 0
 }
 
-// Add inserts an item, and puts it in the queue. The item is only enqueued
-// if it doesn't already exist in the set.
+// Add inserts an item to `items`, and puts it in the `queue`. The item is only enqueued
+// if it doesn't already exist in `items`.
```

Comment:
I think the existing comment was intentionally not referencing private member variables?

Reply:
Makes sense. Reveted.